server/views/layout.njk

```diff
@@ -75,6 +75,9 @@
   {% include 'partials/licenceDetailsBanner.njk' %}
   {% include 'partials/formError.njk' %}
   <span class="govuk-visually-hidden" id="{{ pageId }}"></span>
+
+  {# IE issue where keyboard navigation does not tab into main content #}
+  <div id="main" tabindex="-1"></div>
```

Comment:
Not familiar with this issue

Reply:
Its a funny one where IE does not focus the main window without a mouse-click into it.  This is one suggested solution, and the other is an empty link `<a id="main" name="main" tabindex="-1">` - neither actually do anything, but apparently make some versions of IE allow tab into the main content. Experimental, as marked! Trouble is I can't test this locally :-(